corehq/apps/hqwebapp/views.py

```diff
@@ -395,6 +395,19 @@ def debug_notify(request):
             "If you want to achieve a 500-style email-out but don't want the user to see a 500, use notify_exception(request[, message])")
     return HttpResponse("Email should have been sent")
 
+
+@require_POST
+def jserror(request):
+    notify_js_exception(
+        request,
+        message=request.POST.get('message', None),
+        stack=request.POST.get('stack', None),
+        line=request.POST.get('line', None),
+        filename=request.POST.get('filename', None),
+    )
+    return HttpResponse("JS error has been logged")
```

Comment:
Very minor, but might want to consider sending the most basic response possible here. I'm thinking something along the lines of 200 OK with no content. Anyone who hits this handler knows they're submitting a JS error, and reducing bandwidth overhead in any way possible seems ideal.